prac_06/languages.py

```diff
@@ -0,0 +1,17 @@
+"""
+Languages.
+Estimate: 20 mins 4:59pm
+Actual: 32 mins 5:31pm
+"""
+
+from prac_06.programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+print(python)
+programming_languages = [python, ruby, visual_basic]
```

Comment:
Do these need to be variables as well as a list?

Reply:
The objects could be created while crating the list variable. However I've decided to leave it this way because I believe the task has asked us to use the existing objects.